schema-utils/src/main/scala/uk/gov/nationalarchives/tdr/schemautils/SchemaUtils.scala

```diff
@@ -32,4 +32,9 @@ object SchemaUtils {
       .map(_.getKey)
       .toList
   }
+
+  def getPropertyField(propertyKey: String, fieldName: String): JsonNode = {
```

Comment:
Can you use the path to get any field value because it won't work with nested fields?

` def getPropertyValue(path: String): JsonNode = {
    schemaNode.at(s"/properties/$path")
  }
`
To call: `getPropertyValue("file_path/propertyType").`